policy/cc-license-odrl-map.json

```diff
@@ -0,0 +1,86 @@
+{
+  "CC0": {
+    "@context": "http://www.w3.org/ns/odrl.jsonld",
+    "@type": "Set",
+    "uid": "http://example.com/policy:cc0",
+    "permission": [{"action": "use"}]
+  },
+  
+  "CC BY": {
+    "@context": "http://www.w3.org/ns/odrl.jsonld",
+    "@type": "Set",
+    "uid": "http://example.com/policy:cc-by",
+    "permission": [{
+      "action": "use",
+      "duty": [{"action": "attribute"}]
+    }]
+  },
+  "CC BY-SA": {
+    "@context": "http://www.w3.org/ns/odrl.jsonld",
+    "@type": "Set",
+    "uid": "http://example.com/policy:cc-by-sa",
+    "permission": [{
+      "action": "use",
+      "duty": [
+        {"action": "attribute"},
+        {"action": "shareAlike"}
```

Comment:
When considering this as RDF, what namespace is `shareAlike` in? I see that ODRL, here, relies on the CC namespace (https://www.w3.org/TR/odrl-vocab/#term-ShareAlike).